modules/tinymce/src/core/main/ts/api/dom/Selection.ts

```diff
@@ -57,6 +62,8 @@ const isValidRange = (rng: Range) => {
 interface EditorSelection {
   bookmarkManager: BookmarkManager;
   controlSelection: ControlSelection;
+  _tableResizeHandler: TableResizeHandler;
+  _tableCellSelection: TableCellSelection;
```

Comment:
I'm not totally comfortable leaving these on the interface, even if they are underscore prefixed. We might need to look at a better way before the feature branch is merged.

Reply:
I have created a Jira and linked it to the Epic to investigate this.